src/aktualizr_info/main.cc

```diff
@@ -77,219 +78,237 @@ int main(int argc, char **argv) {
       readonly = false;
     }
 
+    bool wait_provision_flag = false;
+    if (vm.count("wait-until-provisioned") != 0) {
+      wait_provision_flag = true;
+    }
+
     std::shared_ptr<INvStorage> storage = INvStorage::newStorage(config.storage, readonly);
     bool cmd_trigger = false;
+    bool provision_trigger = true;
```

Comment:
Might be simpler to initialize this to false and just use a `while(!provisioned)` loop.